lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -319,7 +327,11 @@ def _user_grade_results(self, course_grade, context):
         """
         grade_results = []
         if settings.FEATURES.get('ENABLE_GRADE_CONVERSION'):
-            final_grade = "{0:.2f}".format(course_grade.percent / 0.20)
+            total = 0
+            for key, value in course_grade.grader_result['grade_breakdown'].items():
+                percent = _unround_grade(value['percent'])
+                total += percent
+            final_grade = "{0:.2f}".format(total / 0.20)
```

Comment:
Where is the 0.20 coming from?

Reply:
0.20 was already. We did it to convert the percentage note to scale from 0 to 5, but it is controlled by a feature of the env.json